scripts/release.sh

```diff
@@ -12,14 +12,8 @@ function release {
   VERSION="$(get_cabal_version "${NAME}")"
 
   pushd "${NAME}"
-  cabal v1-configure --disable-tests --disable-benchmarks
-  cabal v1-sdist
-  cabal upload --publish "dist/${NAME}-${VERSION}.tar.gz"
-  if [ "${NAME}" = "dhall-nix" ]
-  then
-    cabal v1-haddock --builddir=docs --for-hackage --haddock-options="--hyperlinked-source --quickjump"
-    cabal upload --documentation --publish "docs/${NAME}-${VERSION}-docs.tar.gz"
-  fi
+  nix-shell --run 'cabal configure --disable-tests --disable-benchmarks && cabal sdist'
```

Comment:
Why is the `configure` step necessary? I'm wondering whether it could be skipped. 

```suggestion
  nix-shell --run 'cabal sdist'
```